packages/qwik/src/prefetch-service-worker/direct-fetch.ts

```diff
@@ -45,7 +51,7 @@ function getResponse(swState: SWState, url: URL): Promise<Response> {
 
 async function enqueueFetchIfNeeded(swState: SWState, url: URL, priority: number) {
   let task = swState.$queue$.find((task) => task.$url$.pathname === url.pathname);
-  const mode = priority >= DIRECT_PRIORITY ? 'direct' : 'prefetch';
+  const mode = priority > HIGH_PRIORITY ? 'direct' : 'prefetch';
```

Comment:
why no longer `>=`?

Reply:
I'm testing things to solve the issue #6602